kite-web-element/src/components/filesys/QuickUpload.vue

```diff
@@ -46,17 +46,7 @@ export default {
       return this.$confirm(`delete ${file.name}?`);
     },
     async handleUpload(args) {
-      let req = await this.$uploadFiles({
-        query: this.$gql`
-          mutation($files: [Upload!]!){
-            multipleUpload(userid: "001416358", files: $files){
-                name
-                size
-            }
-          }
-        `,
-        files: [args.file]
-      });
+      let req = await this.$fileManager.uploadFiles('001416358', [args.file]);
```

Comment:
Can this code be dynamic now or does that require additional work?

Reply:
Referencing the uploadFiles('001416358') part

Reply:
From my understanding of the mutation i wrote (if it's the same) then all he needs to do is access the vuex store for the user id

Reply:
@ArowShot

Reply:
Fixed, also it wouldn't affect anything we don't use the component